Source/Analytics/Read/Map/CaseReportsBeforeDayQuery.cs

```diff
@@ -0,0 +1,18 @@
+using System.Linq;
+using Dolittle.Queries;
+using Dolittle.ReadModels;
+using Concepts;
+
+namespace Read.Map
+{
+    public class CaseReportsBeforeDayQuery : IQueryFor<CaseReportsBeforeDay>
```

Comment:
Comment from Gard on other PR: "Same as the other here, what about CaseReportsForMapQuery ? We only want to get the last day anyway (I think 😅), so I don't see a reason to include day anywhere in the naming."